src/mappings/staking.ts

```diff
@@ -51,19 +51,6 @@ import {
 } from './helpers/helpers'
 import { addresses } from '../../config/addresses'
 
-export function handleDelegationParametersUpdated(event: DelegationParametersUpdated): void {
-  let graphNetwork = createOrLoadGraphNetwork(event.block.number, event.address)
-  let indexer = createOrLoadLegacyIndexer(event.params.indexer, event.block.timestamp)
-  indexer.indexingRewardCut = event.params.indexingRewardCut.toI32()
-  indexer.queryFeeCut = event.params.queryFeeCut.toI32()
-  indexer.delegatorParameterCooldown = event.params.cooldownBlocks.toI32()
-  indexer.lastDelegationParameterUpdate = (
-    addresses.isL1 ? event.block.number : graphNetwork.currentL1BlockNumber!
-  ).toI32()
-  indexer = updateAdvancedIndexerMetrics(indexer as Indexer)
-  indexer.save()
-}
-
```

Comment:
I would not delete this at all, since we essentially lose historical tracking of delegation cuts. Particularly given that we are now moving all of the new delegation cuts to line up with these ones 😅